frontend/packages/helm-plugin/integration-tests/features/helm/helm-installation-view.feature

```diff
@@ -0,0 +1,38 @@
+@helm
+Feature: Helm Chart Installation View
+              As a user, I should be able switch between YAML and Form view to install Helm Chart
```

Comment:
Indentation

Reply:
Updated automatically on save and not able to see any gherkin-lint issue